go/vt/vtgate/planbuilder/testdata/unsupported_cases.json

```diff
@@ -358,5 +358,10 @@
     "comment": "WITH ROLLUP not supported on sharded queries",
     "query": "select a, b, c, sum(d) from user group by a, b, c with rollup",
     "plan": "VT12001: unsupported: GROUP BY WITH ROLLUP not supported for sharded queries"
+  },
+  {
+    "comment": "WITH ROLLUP not supported on sharded queries",
```

Comment:
This comment seems off?

Reply:
![image](https://github.com/vitessio/vitess/assets/402248/813808cf-2f1d-4bf2-a3cd-1f9ff11bcfb5)